public/index.html

```diff
@@ -2,14 +2,23 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="./logo.svg" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="logo192.png" />
+    <link rel="apple-touch-icon" href="./logo.svg" />
+    <meta
+      name="viewport"
+      content="minimum-scale=1, initial-scale=1, width=device-width"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```

Comment:
Ooh what are these?

Reply:
Great question, @sandaiiyahh! The meta tag there is for responsiveness. Honestly, I don't know if there's much of a difference from the meta tag we replaced, but this is what it says [here](https://material-ui.com/getting-started/usage/#responsive-meta-tag):

> Material-UI is developed mobile-first, a strategy in which we first write code for mobile devices, and then scale up components as necessary using CSS media queries. To ensure proper rendering and touch zooming for all devices, add the responsive viewport meta tag to your <head> element.

The links for the google font api I think are remnants of when we were trying to figure out how to import the Asap font and incorporate it with Material-UI. I don't think those lines are necessary because we imported the font in `src/themes/theme.js`. After commenting out those link tags and testing it, the Asap font is still there, so I think I'm going to delete those. 

Thanks for asking those questions! 🙂

Reply:
Ooh, I seee. Thanks for explaining :)